modules/sdk-core/src/v2/environments.ts

```diff
@@ -172,7 +172,8 @@ export const Environments: Environments = {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     uri: process.env.BITGO_CUSTOM_ROOT_URI!,
     network: process.env.BITGO_CUSTOM_BITCOIN_NETWORK as V1Network,
-    hsmXpub: hardcodedPublicKeys.hsmXpub.dev,
+    hsmXpub:
+      process.env.NODE_ENV === 'production' ? hardcodedPublicKeys.hsmXpub.prod : hardcodedPublicKeys.hsmXpub.test,
```

Comment:
Do you want `dev` instead of `test`?

Reply:
I think it should be `test` since clients can use NODE_ENV=development for testing out features in bitgo-test